stdlib/3/os/__init__.pyi

```diff
@@ -35,20 +35,22 @@ if sys.platform != 'win32':
     F_ULOCK: int
     F_TEST: int
 
-    POSIX_FADV_NORMAL: int
-    POSIX_FADV_SEQUENTIAL: int
-    POSIX_FADV_RANDOM: int
-    POSIX_FADV_NOREUSE: int
-    POSIX_FADV_WILLNEED: int
-    POSIX_FADV_DONTNEED: int
+    if sys.platform != "darwin":
```

Comment:
The documentation says "Availability: Unix". I just tested with Python 3.7.6 on Windows 10, and they don't seem to exist.

Reply:
Ignore me, I didn't notice this is inside a `!= "win32"` branch.